po/fr.po

```diff
@@ -523,7 +513,7 @@ msgstr "Trait sombre"
 
 #: plugins/base16/oomox_plugin.py:255
 msgid "Base16 Export Options…"
-msgstr "Options du thème du Base16…"
+msgstr "Options du thème du Base16 …"
```

Comment:
why space before punctuation?

Reply:
Because according to the Metropolitan French grammar, the space is required before punctations:

Êtes-vous sûr ?
No: Êtes-vous sûr?

Je vais étudier !
No: Je vais étudier!

Deux options : Oomox et Papirus Kolorizer
No: Deux options:

Only in Quebec (Canada) where somebody speaks French, the space is not required.

Reply:
today i've learned it, thanks :-)